src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,141 @@
+import random
+from statistics import median, mean, stdev
+
+
+def single_game(num_players):
+    """
+    Returns duration of single game.
+
+    Arguments
+    ---------
+    num_players : int
+        Number of players in the game
+
+    Returns
+    -------
+    num_moves : int
+        Number of moves the winning player needed to reach the goal
+    """
+    num_moves = 0
+    players = create_players(num_players)
```

Comment:
please define functions in the order they are used. e.g. `create_players()` should be defined above the `single_game()` function.